lib/chewy/type/mapping.rb

```diff
@@ -165,46 +165,41 @@ def agg(name, &block)
         #   template template42: {match: 'hello*', mapping: {type: 'object'}} # or even pass a template as is
         #
         def template(*args)
-          build_root.dynamic_template(*args)
+          root.dynamic_template(*args)
```

Comment:
Can't `root_object` be used directly when it is not updated here?

Reply:
Don't like the idea, there should be some DSL. Are you proposing to get rid of this method?